### PR TITLE
Handle errors in `ApiRepoUpdateSlice`

### PR DESCRIPTION
**Pull request by olenagerasimova**
Closes https://github.com/artipie/artipie/issues/320

### CODE REVIEW

File: src/test/java/com/artipie/management/api/ApiRepoUpdateSliceTest.java [hunk 1]
```diff
@@ -124,6 +127,27 @@ void updatesRepoConfiguration(final String extension, final boolean param) {
...
+                    RqMethod.POST, String.format("/api/repos/%s", ApiRepoUpdateSliceTest.USER)
+                ),
+                new Headers.From(
+                    "Location", String.format("/dashboard/%s", ApiRepoUpdateSliceTest.userRepo())
+                ),
+                new Content.From(body(ApiRepoUpdateSliceTest.REPO, "abc123 : sdfdsf : sdfsdf"))
+            )
+        );
+    }
+
```
Comment (genryxy) [inline src/test/java/com/artipie/management/api/ApiRepoUpdateSliceTest.java:150]: @olenagerasimova thanks, one suggestion. Probably to add negative test for `INTERNAL_ERROR`?

**Comment by olenagerasimova** [comment]
@genryxy thanks, added

**Review by genryxy — APPROVED** [review]
@olenagerasimova thanks, looks good to me

**Comment by olenagerasimova** [comment]
@g4s8 please, take a look/merge

**Pull request merged**